### ISSUE TITLE
Style/FormatStringToken false positive for double %

### ISSUE THREAD

**Issue by svyatov** [rubocop/rubocop#7439]
Rubocop sees `Style/FormatStringToken` in escaped `%` sign.

```ruby
format '%<hit_rate>6.2f%%', hit_rate: 12.34
# => " 12.34%"
```

--------

## Expected behavior

Rubocop must not recognize `Style/FormatStringToken` offense here.

## Actual behavior

```
Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over unannotated tokens (like %s).
        format '%<hit_rate>6.2f%%', hit_rate: hit_rate
                               ^^
```

## Steps to reproduce the problem

Use provided code sample.

## RuboCop version

```
$ [bundle exec] rubocop -V
0.75.1 (using Parser 2.6.5.0, running on ruby 2.6.1 x86_64-darwin18)
```

**Comment by buehmann** [comment]
Thanks for the report! I am taking of this.
